package.json

```diff
@@ -1,7 +1,16 @@
 {
   "name": "@observablehq/database-proxy",
   "description": "A local proxy to connect private Observable notebooks to private databases",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
actually, leave the version unchanged for now. we can bump if if/when we actually make a release.